src/Group.php

```diff
@@ -25,7 +25,7 @@ final class Group implements GroupInterface
     private bool $routesAdded = false;
     private bool $middlewareAdded = false;
     private array $disabledMiddlewareDefinitions = [];
-    private array $autoOptions = [];
+    private array $corsMiddlewares = [];
```

Comment:
Do we need an array if `withCors()` now accepts a single middleware?

Reply:
Yes `withCors()` now accepts a single middleware definition but, can be used multiple times `withCors()->withCors()`...

Reply:
Any scenario there more than one CORS middleware needed? just asking

Reply:
I don't have any.